tests/base/utilities_01.cc

```diff
@@ -65,6 +65,18 @@ void test ()
                  ExcInternalError());
   }
 
+  {
+    const char *p = "alpha;; beta;; gamma ";
+    AssertThrow (Utilities::split_string_list (p, ";;").size() == 3,
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[0] == "alpha",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[1] == "beta",
+                 ExcInternalError());
+    AssertThrow (Utilities::split_string_list (p, ";;")[2] == "gamma",
+                 ExcInternalError());
```

Comment:
I'm a big fan of using Assert also in tests. It makes it so much easier to debug tests than if only the output is wrong. It also avoids the need to store (possibly large) output. In other words, if you know what the exact answer is, I think Assert is the way to go.